browse/static/js/catalyzex.js

```diff
@@ -0,0 +1,64 @@
+(async () => {
+  const arxivId = window.location.pathname.match(/(?:abs|pdf)\/(.+\d{3})/)[1];
+  const paperTitle = document.querySelector("h1.title")?.innerText;
+  const paperUrl = window.location.href;
```

Comment:
to remove any queryStrings at the end of URL (e.g. `?test=foo`)
```suggestion
  const paperUrl = window.location.href.split('?')[0];
```

Reply:
Done :)